### ISSUE TITLE
Convert TensorFlow .pb file to Keras .h5 file

### ISSUE THREAD

**Issue by ryspark** [orangese/facial-recognition#1]
@orangese @22pilarskil 

We're currently using a pretrained model from https://github.com/nyoki-mtl/keras-facenet. This model was trained on MS-CELEB-1M and achieved 99.4% LFW accuracy. 

However, this repository (https://github.com/davidsandberg/facenet) has a model with the same architecture trained on VGGFace2 (3 million images) that achieved 99.65% LFW accuracy. It is therefore preferable to the Keras FaceNet that we're currently using. The problem with using this model is that it is provided as .pb and .meta files, which are compatible with TensorFlow but not Keras.

 It would be great if someone could convert the TensorFlow FaceNet model into a Keras model (stored as a .h5 file).

TensorFlow model: https://drive.google.com/file/d/1EXPBSXwTaqrSC0OhUdXNmKSh9qJUQ55-/view

**Comment by ryspark** [comment]
Moving issue to TODO.md file.
